staging/src/k8s.io/apiserver/pkg/endpoints/watch_test.go

```diff
@@ -757,6 +757,62 @@ func TestWatchHTTPTimeout(t *testing.T) {
 func BenchmarkWatchHTTP(b *testing.B) {
 	items := benchmarkItems(b)
 
+	// benchmark may have fuzzed non unicode names, normalize for this test to the ASCII set
```

Comment:
These two tests seem to differ only by this normalization - can we extract the test to a function to avoid code duplication?
Sth like:
```
items := benchmarkItems(b)
// Normalize if needed (depending on the test)
benchmarkWatch(b, items)
```